backend/application/syson-application/src/test/java/org/eclipse/syson/application/libraries/publication/SySONLibraryPublicationTests.java

```diff
@@ -10,7 +10,7 @@
  * Contributors:
  *     Obeo - initial API and implementation
  *******************************************************************************/
-package org.eclipse.syson.application.publication;
+package org.eclipse.syson.application.libraries.publication;
```

Comment:
I moved this test in `libraries/publication` to uniformize the package structure w.r.t. import and update